prismic-model/js/collection-venues.js

```diff
@@ -0,0 +1,109 @@
+import description from './parts/description';
+import timestamp from './parts/timestamp';
+import number from './parts/number';
+const CollectionVenue = {
+  Main: {
+    title: {
+      type: 'Text',
+      config: {
+        label: 'Title'
+      }
+    },
+    description,
+    order: number('Order')
```

Comment:
Why don't we do this on the programmatic side?
The fact we're looking at bundling two together makes me think that's the place for it to live.

Reply:
I had this in code before and you commented:

> Why not store this in Prismic, and then use it as an ordering?

Maybe I misunderstood. It makes it more flexible being in Prismic, bu tit might be overkill

Reply:
How are you going to deal with the bundling of Galleries & Library and eaty things?